### PR TITLE
update docs for the new AEPMobileServices extension

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: using-mobile-extensions/adobe-analytics-mobile-services/README.md [hunk 1]
```diff
@@ -291,6 +363,28 @@ Use the following API to track a push messaging click through in Adobe Analytics
 }
 ```
 {% endtab %}
+
+{% tab title="iOS — Swift" %}
+Use the following API to track a push messaging click through in Adobe Analytics.
```
Comment (ahmyr) [inline using-mobile-extensions/adobe-analytics-mobile-services/README.md:368]: Take out "through", so it is "click in Adobe Analytics".

Comment (yangyansong-adbe) [reply using-mobile-extensions/adobe-analytics-mobile-services/README.md]: fixed

**Review by nporter-adbe — APPROVED** [review]

File: using-mobile-extensions/adobe-analytics-mobile-services/migration.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Migration to AEPMobileServices
+
+This document is a reference comparison of ACPMobileServices\(1.x\) APIs against their equivalent APIs in AEPMobileServices\(3.x\).
+
+## Primary `Classes`
+
+The class name containing public APIs is different depending on which SDK and language combination being used.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/migration.md:7]: "... on which SDK and language combination is being used."

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ String mobileServicesExtensionVersion = MobileServices.extensionVersion();
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS — Obj-C" %}
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:18]: Same comment about Objective-C.

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 4]
```diff
@@ -28,6 +28,22 @@ NSString *mobileServicesExtensionVersion = [ACPMobileServices extensionVersion];
 let mobileServicesExtensionVersion  = ACPMobileServices.extensionVersion()
 ```
 {% endtab %}
+
+{% tab title="iOS — Swift" %}
+**Objective C**
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:33]: Should be Objective-C, not Objective C. (This goes for all instances in this doc and other docs.)

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 5]
```diff
@@ -58,7 +74,7 @@ Uri testUri = new Uri.Builder()
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS — Obj-C" %}
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:77]: Same comment.

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 6]
```diff
@@ -82,6 +98,34 @@ let url = URL(string: "adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.
...
+
+#### Syntax
+
+```text
++ (void) trackAdobeDeepLink: (NSURL* _Nonnull) deeplink;
+```
+
+#### Example
+
+#### Objective C
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:112]: Same comment about this being "Objective-C".

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 7]
```diff
@@ -82,6 +98,34 @@ let url = URL(string: "adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.
...
+
+```text
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:114]: Shouldn't this be an objectivec for the syntax highlighting?

**Review by calebk1 — COMMENTED** [review]
Looks good! Just a few minor comments.

File: using-mobile-extensions/adobe-analytics-mobile-services/README.md [hunk 8]
```diff
@@ -119,7 +119,7 @@ import com.adobe.marketing.mobileservices.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS — Obj-C" %}
```
[the remark this answers is not in the document]
Comment (yangyansong-adbe) [reply using-mobile-extensions/adobe-analytics-mobile-services/README.md:122]: https://aep-sdks.gitbook.io/docs/foundation-extensions/profile#add-profile-to-your-app
I'd like to not change it for now to keep consistency as we're using the same tab name on Profile docs, I can create a separate PR later to update them all.

File: using-mobile-extensions/adobe-analytics-mobile-services/README.md [hunk 9]
```diff
@@ -120,7 +120,8 @@ import com.adobe.marketing.mobileservices.*;
 {% endtab %}
 
 {% tab title="iOS — Obj-C" %}
-Add the library to your project via your Podfile by adding the `ACPMobileServices`pod.
+
+You can add the library to your project through your `Podfile` by adding the `ACPMobileServices`pod.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/README.md:124]: We need a space between `ACPMobileServices` and pod.

File: using-mobile-extensions/adobe-analytics-mobile-services/README.md [hunk 10]
```diff
@@ -136,7 +137,8 @@ Import the library into your project:
 {% endtab %}
 
 {% tab title="iOS — Swift" %}
-Add the library to your project via your Podfile by adding the `AEPMobileServices`pod.
+
+You can add the library to your project through your `Podfile` by adding the `AEPMobileServices`pod.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-analytics-mobile-services/README.md:141]: Same comment here, we're missing a space between `AEPMobileServices` and pod.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**